package.json

```diff
@@ -6,33 +6,44 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "jest",
+    "lint": "ng lint",
+    "format:fix": "pretty-quick --staged",
+    "prepare": "husky"
   },
   "private": true,
   "dependencies": {
     "@angular/animations": "^19.0.0",
```

Comment:
TS2307: Cannot find module 'openai' or its corresponding type declarations. [plugin angular-compiler]

    src/app/shared/service/chat-bot-ai.service.ts:3:19:
      3 │ import OpenAI from 'openai';
        ╵                    ~~~~~~~~

 'openai' are missing in yours package.json

Reply:
https://www.npmjs.com/package/openai

Reply:
"openai": "^4.77.0",